preTrainedSample.py

```diff
@@ -286,8 +286,13 @@ def testModel(testFileName, modelTrained, classLabels):
     """
     contents = open(testFileName).readlines()
     results = []
+    voc = vectorizer.get_vocabulary(include_special_tokens=False)
+
     for con in contents:
         label, text = con.rstrip().split(',', 1)
+        misses = [word for word in text.split(' ') if word not in voc]
+        if misses:
+            raise ValueError(f"Invalid tokens {', '.join(misses)} given")
```

Comment:
Causing a complete failure by raising an error is too drastic.  Instead, print out the bad word *and* the complete line of text so that the line can be easily found.